images/extension-builders/entrypoint.sh

```diff
@@ -14,25 +14,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -e
+set -ue
 
 SCRIPT_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}")"  && pwd)
 GETENVOY_WORKSPACE_DIR="${GETENVOY_WORKSPACE_DIR:-$PWD}"
```

Comment:
this isn't overridden anywhere, it isn't used in tinygo, and I don't think it helps to alias PWD. Can we get rid of this?

Reply:
for example, since initial import the work dir (pwd) was fixed via this `"-w", "/source",` in toolchain.go

Reply:
looks like we can get rid of.